test/types/list_test.rb

```diff
@@ -53,14 +69,15 @@ class ListTest < ActiveSupport::TestCase
     assert_equal %w[ 2 3 ], @list.last(2)
   end
 
-  test "typed as datetime" do
-    @list = Kredis.list "mylist", typed: :datetime
-
-    @list.append [ 1.day.from_now.midnight.in_time_zone("Pacific Time (US & Canada)"), 2.days.from_now.midnight.in_time_zone("UTC") ]
-    assert_equal [ 1.day.from_now.midnight, 2.days.from_now.midnight ], @list.elements
+  test "size" do
+    @list.clear
```

Comment:
Would `assert_equal 0, @list.size` here rather than having a separate test for size after removal.